docs/modules/ROOT/pages/explanation/decisions/zone-naming.adoc

```diff
@@ -0,0 +1,46 @@
+= Zone naming scheme
+
+== Problem
+
+{product} will consist of several clusters we refer to as zones.
+We need a common understanding how we name them.
+
+== Alternatives
+* Using a randomly generated identifier like `rapid-sound-0933` or `rapid-sound` (`rapid-sound.appuio.cloud`).
+  We have tried that already with https://syn.tools[Project Syn].
+  This is way better than for example a random string of characters or a UUID.
+  From the perspective of daily operation, we have learned that this was still too much mental load even for a small number of items.
+* Use top level domain to encode country locality (`<zone>.appuio.ch`, `<zone>.appuio.de`).
+  This does not work as not every country has a country specific top level domain (for example the United States of America).
+* Use the cloud provider followed by a numerical or alphabetical index (`cloudscale-a.appuio.cloud` or `exoscale-1.appuio.cloud`).
+* Use the cloud provider followed by the zone abbreviation used by the cloud provider (`cloudscale-rma1.appuio.cloud` or `exoscale-ch-gva-2.appuio.cloud`). (See https://www.cloudscale.ch/en/api/v1#regions and https://www.exoscale.com/datacenters/)
+** Optionally shorten the cloud provider name, e.g. `clsc` or `exo`
+* Use the country code followed by a numerical or alphabetical index (`ch-1.appuio.cloud` or `de-a.appuio.cloud`).
+
+== Decision
+
+The zone identifier will be built following the pattern:
+
+****
+`<provider>-<region>-<index>`
+****
+
+`<provider>`::: Name of the cloud provider the cluster is hosted on (`cloudscale`, `exoscale`, `aws`, `azure`, `gcp`, etc.).
+`<region>`::: Region name, taken verbatim from the cloud providers own naming scheme (`rma`, `ch-dk-2`, `eu-central-1`, `switzerlandnorth`, `europe-west6`).
+`<index>`::: Zero indexed numerical identifier.
+
+.List of examples
+====
+* `cloudscale-rma-0.appuio.cloud` first cluster hosted on Cloudscale and placed in the region located in Rümlang.
+* `exoscale-ch-dk-2-2.appuio.cloud` third cluster hosted on Exoscale and placed in the region located in Zürich.
+* `aws-eu-central-1-1.appuio.cloud` second cluster hosted on AWS and placed in the region located in Frankfurt.
+====
+
+[NOTE]
+====
+On Cloudscale, you will most likely only see `rma1` and `lpg1`.
+One is inclined to assume those are the regions, but actually, those are the zones.
+As Cloudscale only has one zone per region, it is easy to confuse one for the other.
+====
+
+== Rationale
```

Comment:
I'm a bit unhappy with the formulation/structure of this decision.

The way Alternatives is initially meant to be structured is that they don't contain random partial ideas, but actual, complete proposals where one could choose one of. The naming scheme described currently in the Decision section isn't listed at all in the Alternatives section but instead combines several ideas from the Alternatives into one. The Decision section could just contain one line: Which alternative was choosen. Probably makes sense to label the Alternatives or at least number them, so we can say "We've choosen Option 2"

The WHY belongs to Rationale which is currently completely empty. 

Examples of how Cloudscale naming schemes could look like belong to the Alternatives IMO.

For example, listing `Use top level domain to encode country locality (<zone>.appuio.ch, <zone>.appuio.de)` isn't a complete Proposal as the values of `<zone>` is undefined.

Options that were never really on the table should be removed completely. I'd rather have 2 complete Alternatives than 10 ideas.

Reply:
As discussed in-person, we will not document the naming scheme as an architecture decision since the naming scheme has no relevant impact on the architecture.
Instead we simply document it as a definition in the reference.